src/main/pages/che-7/overview/proc_importing-projects-from-github-in-hosted-che.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+// This module can be included from assemblies using the following include statement:
+// include::<path>/proc_importing-projects-from-github-in-hosted-che.adoc[leveloffset=+1]
+
+// The file name and the ID are based on the module title. For example:
+// * file name: proc_doing-procedure-a.adoc
+// * ID: [id='proc_doing-procedure-a_{context}']
+// * Title: = Doing procedure A
+//
+// The ID is used as an anchor for linking to the module. Avoid changing
+// it after the module has been published to ensure existing links are not
+// broken.
+//
+// The `context` attribute enables module reuse. Every module's ID includes
+// {context}, which ensures that the module has a unique ID even if it is
+// reused multiple times in a guide.
+//
+// Start the title with a verb, such as Creating or Create. See also
+// _Wording of headings_ in _The IBM Style Guide_.
+[id="importing-projects-from-github-in-hosted-che_{context}"]
+= Importing projects from GitHub in Hosted Che
+
+This section describes how to import a GitHub project in Hosted Che.
+
+[discrete]
+== Prerequisites
+
+* Navigate to the *Projects* section of the *New Workspace* creation screen. 
```

Comment:
* This sounds more like step #1
* Isn't the procedure about linking accounts a prereq here?

Reply:
OK done.